rook/README.md

```diff
@@ -0,0 +1,27 @@
+[![Docker Repository on Quay](https://quay.io/repository/cybozu/rook/status "Docker Repository on Quay")](https://quay.io/repository/cybozu/rook)
+
+Rook container
+==============
+
+This directory provides a Dockerfile to build a rook container that runs [rook][].
+
+Because of the following reasons, the Dockerfile uses [cybozu-go/rook:release][].
+1. The current version of Rook (v1.2.2) does not support partition (`part`) device type, though `master` branch already supported it. So we need to use `master` branch source code.
+2. The current version of Rook (v1.2.2) does not support `topologySpreadConstraints` feature. So we need to patch the original code of Rook.
+
```

Comment:
It looks a bit verbose. So, how about the following description.

```
This container uses a cybozu's own Rook [cybozu-go/rook:release][]. It's because the following features are necessary for Neco and are not in the upstream's newest stable version (v1.2.2).

* partition device type (blkid's "part" type):  This is supported in `master`.
* topologySpreadConstraints`: We created a branch to support it. However, merging it to upstream Rook would take time (see Rook's issue XX).

Our custom Rook is created as follows to support the above-mentioned features.
```